package.json

```diff
@@ -3,15 +3,19 @@
   "version": "1.0.0",
   "description": "",
   "scripts": {
-    "start": "npm run build && node server.js",
+    "start": "npm run build && npm run server",
+    "dev": "npm-run-all --parallel server watch:**",
     "build": "npm-run-all build:**",
     "build:clean": "rimraf ./build && mkdirp ./build",
-    "build:static": "ncp ./static ./build",    
+    "build:static": "ncp ./static ./build",
     "build:css": "autoless --no-watch --autoprefix \"> 5%, last 2 versions, android >= 4.0\" ./src ./build/css",
     "build:js": "webpack",
+    "watch:js": "webpack --watch",
+    "watch:css": "autoless --autoprefix \"> 5%, last 2 versions, android >= 4.0\" ./src ./build/css",
```

Comment:
This is pretty non-dry with the `build:css` script above. Take a look at https://github.com/mozilla/webmaker-android/blob/develop/package.json for example of how to thin this out.